Source/ConfigCat/Private/ConfigCatSubsystem.cpp

```diff
@@ -279,7 +284,12 @@ void UConfigCatSubsystem::Initialize(FSubsystemCollectionBase& Collection)
 	const UConfigCatSettings* ConfigCatSettings = GetDefault<UConfigCatSettings>();
 	if (!ConfigCatSettings || ConfigCatSettings->SdkKey.IsEmpty())
 	{
-		UE_LOG(LogConfigCat, Warning, TEXT("Empty SdkKey detected. Please set your SdkKey in the Project Settings."));
+		if(GEngine)
```

Comment:
```suggestion
		if (GEngine)
```

Reply:
I ran the formatter now and uncovered more of these cases, they are now all part of this PR